ui/app/templates/vault/cluster.hbs

```diff
@@ -1,9 +1,11 @@
 <Sidebar::Nav::Cluster />
-
-<LicenseBanners
-  @expiry={{this.activeCluster.licenseExpiry}}
-  @autoloaded={{eq this.activeCluster.licenseState "autoloaded"}}
-/>
+{{! Only show license banners for Enterprise }}
+{{#unless this.activeCluster.version.isOSS}}
+  <LicenseBanners
```

Comment:
Just for my knowledge (no need to update) - we could also check {{#if this.activeCluster.version.isEnterprise}} instead of using {{#unless this.activeCluster.version.isOSS}} here? I always get confused on unless logic so I wanted to confirm my understanding! 😊

Reply:
Good catch. Honestly, my brain was on fire after working through the logic on this issue and reproducing. I just missed this. Fixed.